<ui version="4.0" >
 <class>Mesh_segmentation_dialog</class>
 <widget class="QDialog" name="Mesh_segmentation_dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mesh segmentation parameters</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="Number_of_rays_label" >
       <property name="text" >
        <string>Square root of number of rays:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="Number_of_rays_spin_box" >
       <property name="value" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="Cone_angle_label" >
       <property name="text" >
        <string>Cone angle:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="Cone_angle_spin_box" >
       <property name="maximum" >
        <number>180</number>
       </property>
       <property name="value" >
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="Number_of_clusters_label" >
       <property name="text" >
        <string>Number of clusters:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="Number_of_clusters_spin_box" >
       <property name="value" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="Smoothness_label" >
       <property name="text" >
        <string>Smoothness:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QDoubleSpinBox" name="Smoothness_spin_box" >
       <property name="singleStep" >
        <double>0.500000000000000</double>
       </property>
       <property name="value" >
        <double>15.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SDF_button" >
       <property name="text" >
        <string>Caluclate SDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Partition_button" >
       <property name="text" >
        <string>Partition</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
